src/main/resources/net/sf/jabref/gui/keyboard/KeyBindingsDialog.css

```diff
@@ -0,0 +1,12 @@
+.keybinding-table {
+	-fx-padding: 5px;
```

Comment:
Try if disabling the grid line (the separator between the columns) with ` -fx-table-cell-border-color: transparent;` looks good.